### DISCUSSION TITLE
Предлогаю добавить настройку стартового роута

### DISCUSSION THREAD

**Discussion by khlystou** [moonshine-software/moonshine#381] [Ideas]
Предлогаю добавить роут в настройки, на который будет вести после успешной авторизации
![image](https://github.com/moonshine-software/moonshine/assets/17690542/8abef6a3-2395-4e9f-87bd-dc0dd26d6596)

**Comment by khlystou** [comment]
Если я правильно понимаю, то этот роут используется только в AuthenticateController & SocialiteController. Могу закрыть эту задачу.

**Reply by lee-to** [reply]
Да сделай pr! Уже все готово у тебя

**Reply by khlystou** [reply]
Готово: https://github.com/moonshine-software/moonshine/pull/387
